internal/validation/validate_cluster.go

```diff
@@ -371,6 +371,12 @@ func validateUserAssignedIdentitiesProfile(ctx context.Context, op operation.Ope
 		newObj.ControlPlaneOperators, safe.Field(oldObj, toPlatformOperatorsAuthenticationUserAssignedIdentitiesControlPlaneOperators),
 		validate.RequiredValue,
 	)...)
+	// even though it's not listed, prior validation had the value required.
```

Comment:
quirk of old validation.  Having a format for an optional field is definitely a thing in general.